sections/NeoVM_NeoContract.md

```diff
@@ -0,0 +1,113 @@
+# NeoVM and NeoContract
```

Comment:
Maybe we should separate NeoVM from NeoContract.

Reply:
sounds good